homeassistant/components/venstar/climate.py

```diff
@@ -257,7 +276,12 @@ def _set_operation_mode(self, operation_mode):
             _LOGGER.error("Failed to change the operation mode")
         return success
 
-    def set_temperature(self, **kwargs):
+    async def async_set_temperature(self, **kwargs):
```

Comment:
We don't need to add the async api. The library doesn't support asyncio. We can call `self.schedule_update_ha_state` to schedule an entity state update from the sync api.

Reply:
But we don't want to schedule an update from the api.  The API already has the correct values, because it records changes in it's internal state. We just want HA to notice that the states might be different now.

Reply:
Yes, that's what `self.schedule_update_ha_state` does.

Reply:
When I was talking about the sync api, I meant the sync api of Home Assistant.

Reply:
ok, then I don't even need the _set_temperature rename at all then, right?  I can just call that at the end?

Reply:
Yes, exactly. It's a smaller change.

Reply:
FWIW, I prefer the call to `async_write_ha_state` since `schedule_update_ha_state` results in additional context switch